lib/structures/Attachment.js

```diff
@@ -4,16 +4,17 @@ const Base = require("./Base");
 
 /**
  * Represents an attachment
- * @prop {id} id The attachment ID
- * @prop {string} filename The filename of the attachment
- * @prop {string?} description The description of the attachment
- * @prop {string?} contentType The content type of the attachment
- * @prop {number} size The size of the attachment
- * @prop {string} url The URL of the attachment
- * @prop {string} proxyUrl The proxy URL of the attachment
- * @prop {string?} height The height of the attachment
- * @prop {string?} width The width of the attachment
- * @prop {boolean?} ephemeral Whether the attachment is ephemeral
+ * @prop {string} data The attachment data
+ * @prop {id} data.id The attachment ID
+ * @prop {string} data.filename The filename of the attachment
+ * @prop {string?} data.description The description of the attachment
+ * @prop {string?} data.contentType The content type of the attachment
+ * @prop {number} data.size The size of the attachment
+ * @prop {string} data.url The URL of the attachment
+ * @prop {string} data.proxyUrl The proxy URL of the attachment
+ * @prop {string?} data.height The height of the attachment
+ * @prop {string?} data.width The width of the attachment
+ * @prop {boolean?} data.ephemeral Whether the attachment is ephemeral
```

Comment:
```suggestion
 * @prop {string} data The attachment data
 * @prop {id} data.id The attachment ID
 * @prop {String} data.filename The filename of the attachment
 * @prop {String?} data.description The description of the attachment
 * @prop {String?} data.contentType The content type of the attachment
 * @prop {Number} data.size The size of the attachment
 * @prop {String} data.url The URL of the attachment
 * @prop {String} data.proxyUrl The proxy URL of the attachment
 * @prop {String?} data.height The height of the attachment
 * @prop {String?} data.width The width of the attachment
 * @prop {Boolean?} data.ephemeral Whether the attachment is ephemeral
 ```

Reply:
`id` isn't a type. And the `data` object is not a `string`

Reply:
> `id` isn't a type. And the `data` object is not a `string`

Those were his 🤷‍♂️